app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.js

```diff
@@ -44,9 +44,9 @@ module.exports = angular.module('spinnaker.core.pipeline.config.pipelineConfigur
     pipelineConfigService.getHistory($scope.pipeline.id, 2).then(history => {
       if (history && history.length > 1) {
         $scope.viewState.hasHistory = true;
-        this.setViewState({ hasHistory: true });
+        this.setViewState({ hasHistory: true, loadingHistory: false });
```

Comment:
if `history.length === 0` the loading indicator will never go away

Reply:
you have a point

Reply:
converted to a `finally` below